src/shaders/fill_extrusion_depth.vertex.glsl

```diff
@@ -0,0 +1,49 @@
+uniform mat4 u_matrix;
+
+attribute vec4 a_pos_normal_ed;
+attribute vec2 a_centroid_pos;
+
+#pragma mapbox: define highp float base
+#pragma mapbox: define highp float height
+
+varying highp float v_depth;
+
+void main() {
+    #pragma mapbox: initialize highp float base
+    #pragma mapbox: initialize highp float height
+
+    vec3 pos_nx = floor(a_pos_normal_ed.xyz * 0.5);
+    // The least significant bits of a_pos_normal_ed.xy hold:
+    // x is 1 if it's on top, 0 for ground.
+    // y is 1 if the normal points up, and 0 if it points to side.
+    // z is sign of ny: 1 for positive, 0 for values <= 0.
+    mediump vec3 top_up_ny = a_pos_normal_ed.xyz - 2.0 * pos_nx;
+
+    float x_normal = pos_nx.z / 8192.0;
```

Comment:
nit: I supposed that it's the tile extent? I believe we have a define for `8192.0` to `EXTENT`, just to suggest what this value represents.